Remotion/Core/Reflection.UnitTests/MethodInfoAdapterTest.cs

```diff
@@ -60,6 +60,18 @@ public void Create_ReturnsSameInstance ()
       Assert.That (MethodInfoAdapter.Create (_method), Is.SameAs (MethodInfoAdapter.Create (_method)));
     }
 
+    [Test]
+    public void CreateOrNull_ReturnsSameInstance ()
+    {
+      Assert.That (MethodInfoAdapter.CreateOrNull (_method), Is.SameAs (MethodInfoAdapter.CreateOrNull (_method)));

```

Comment:
@iam-mholle Reminder: separate issue for the CreateOrNull API when merging to trunk